io_scene_godot/export_godot.py

```diff
@@ -234,7 +365,7 @@ def load_supported_features(self):
                 "Not export to Godot project dir, disable all beta features.")
 
         # minimal supported version
-        conf_versiton = 3
+        conf_version = 3
```

Comment:
Normally it'd be good ettiquette to do typo-fixes as a separate PR (if it was split to a separate PR, I'd merge it straight away) But if you want to leave it here, that should be fine.